android-json-form-wizard/src/main/java/com/vijay/jsonwizard/filesource/AssetsFileSource.java

```diff
@@ -26,7 +27,7 @@ public class AssetsFileSource implements FormFileSource {
     private MVELRuleFactory mvelRuleFactory;
 
     private AssetsFileSource() {
-        this.mvelRuleFactory = new MVELRuleFactory(new YamlRuleDefinitionReaderExt());
+            this.mvelRuleFactory = new MVELRuleFactory(new YamlRuleDefinitionReaderExt());
```

Comment:
Kindly make this optional by switching to `YamlRuleDefinitionReader` or `YamlRuleDefinitionReaderExt`